### PR TITLE
:zap: Release 2.6.3

### PR DESCRIPTION
**Pull request by flovilmart**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+* LRU cache is not available on the ParseServer object, thanks to [Tyler Brock](https://github.com/tbrock)
+* Error messages are more expressive, thanks to [Tyler Brock](https://github.com/tbrock)
+* Postgres: Properly handle undefined field values, thanks to [Diamond Lewis](https://github.com/dlewis)
+* Updating with two GeoPoints fails correctly, thanks to [Anthony Mosca](https://github.com/aontas)
+
+#### New Features:
+* Adds ability to set a maxLimit on server configuration for queries, thanks to [Chris Norris](https://github.com/)
+
+#### Bug fixes:
+* Fixes issue affecting reporting _PushStatus with misconfigured serverURL, thanks to [Florent Vilmart](https://github.com/flovilmart)
```
Comment (acinader) [inline CHANGELOG.md]: hm, is the _ in PushSatus making the rest of it itals??  maybe take out that _

Comment (flovilmart) [reply CHANGELOG.md]: yep you're right!

Comment (acinader) [reply CHANGELOG.md]: according to the interwens, you can escape with \_boo

Comment (acinader) [reply CHANGELOG.md]: ha \\_boo

Comment (flovilmart) [reply CHANGELOG.md]: added backquotes as it looks like code  :)

**Pull request merged**